computer_vision/richardson_lucy.py

```diff
@@ -0,0 +1,241 @@
+"""
+https://en.wikipedia.org/wiki/Motion_blur
+https://en.wikipedia.org/wiki/Image_noise
+https://en.wikipedia.org/wiki/Richardson-Lucy_deconvolution
+"""
+import imageio.v2 as imageio
+import numpy as np
+from numpy.fft import fft2, ifft2, ifftshift
+
+
+def root_mean_square_error(original: np.ndarray, reference: np.ndarray) -> float:
+    """Simple implementation of Root Mean Squared Error
+    for two N dimensional numpy arrays.
+
+    Examples:
+        >>> root_mean_square_error(np.array([1, 2, 3]), np.array([1, 2, 3]))
+        0.0
+        >>> root_mean_square_error(np.array([1, 2, 3]), np.array([2, 2, 2]))
+        0.816496580927726
+        >>> root_mean_square_error(np.array([1, 2, 3]), np.array([6, 4, 2]))
+        3.1622776601683795
+    """
+    return np.sqrt(((original - reference) ** 2).mean())
+
+
+def pad_to_size(image: np.ndarray, reference: np.ndarray):
+    """Pad an image to have final shape equal to reference image."""
+    p, q = (size for size in reference.shape)
```

Comment:
Are these variables necessary? Why not just use `reference.shape[0]` and `reference.shape[1]`?